lib/l10n/app_en.arb

```diff
@@ -951,5 +954,291 @@
+  },
+  "syncDownloads": "Sync and download all items.",
+  "@syncDownloads": {
+    "description": "Tooltip for downloads sync button."
+  },
+  "repairDownloads": "Repair issues with download files or metadata.",
+  "@repairDownloads": {
+    "description": "Tooltip for downloads repair button."
+  },
+  "requireWifiForDownloads": "Require WiFi when downloading.",
+  "@requireWifiForDownloads": {
+    "description": "Description of setting for only starting downloads while on wifi"
+  },
+  "queueRestoreError": "Warning: {count,plural, =1{{count} song} other{{count} Songs}} could not be restored to the queue.",
+  "@queueRestoreError": {
+    "description": "Message that shows when some, but not all, songs in the Now-Playing queue from the previous session cannot be restored due to server issues or being offline.",
+    "placeholders": {
+      "count": {
+        "type": "int"
+      }
+    }
+  },
+  "activeDownloadsListHeader": "{itemCount} {typeName, select, downloading{Running} failed{Failed} syncFailed{Repeatedly Unsynced} enqueued{Queued} other{}} {itemCount, plural, =1{Download} other{Downloads}}}",
+  "@activeDownloadsListHeader": {
+    "description": "Header text for lists of active downloads of a particular state.",
+    "placeholders": {
+      "typeName": {
+        "type": "String"
+      },
+      "itemCount": {
+        "type": "int"
+      }
+    }
+  },
+  "downloadLibraryPrompt": "Are you sure you want to download all contents of the library '{ libraryName }'?",
+  "@downloadLibraryPrompt": {
+    "description": "Confirmation prompt before downloading entire library, which may be large.",
+    "placeholders": {
+      "libraryName": {
+        "type": "String"
+      }
+    }
+  },
+  "onlyShowFullyDownloaded": "Only show fully downloaded albums",
+  "@onlyShowFullyDownloaded": {
+    "description": "Tooltip for button that hides all partially downloaded albums"
+  },
+  "filesystemFull": "Remaining downloads cannot be completed.  The filesystem is full.",
+  "@filesystemFull": {
+    "description": "Message shown when a download cannot complete due to the filesystem being full."
+  },
+  "connectionInterrupted": "Connection interrupted, pausing downloads.",
+  "@connectionInterrupted": {
+    "description": "Message shown when the connection is interrupted during a download."
+  },
+  "connectionInterruptedBackground": "Connection was interrupted while downloading in the background.  This can be caused by OS settings.",
+  "@connectionInterruptedBackground": {
+    "description": "Message shown when the connection is interrupted during a download while in the background.  It advises checking OS settings, as they may cause background downloads to be blocked."
+  },
+  "connectionInterruptedBackgroundAndroid": "Connection was interrupted while downloading in the background.  This can be caused by enabling 'Enter Low-Priority State on Pause' or OS settings.",
```

Comment:
I can increase the timeout pretty easily.  If this pops up frequently though, that could get irritating because the snackbars sometimes block things.

Reply:
Yeah, that's true. Those need to be redesigned asap. Keep the messages as they are :)  
My plan is to have some section (maybe as part of the logs screen) that can show specific messages and errors for later and detailed viewing.